modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-web-console_{context}"]
+= Installing the {TempoShortName} by using the OpenShift web console
+
+You can install the {TempoShortName} in the OpenShift web console.
+
+.Prerequisites
+
+* You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role.
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+.Procedure
+
+. Install the {TempoOperator}:
+
+.. In the *Administrator* view of the OpenShift web console, go to *Operators* -> *OperatorHub* and search for `{TempoOperator}`.
+
+.. Select *{TempoOperator}* that is *OpenShift Operator for Tempo* -> *Install* -> *Install* -> *View Operator*.
++
+[IMPORTANT]
+====
+This installs the Operator with the default presets:
+
+* *Update channel* -> *stable*
+* *Installation mode* -> *All namespaces on the cluster*
+* *Installed Namespace* -> *openshift-tempo-operator*
+* *Update approval* -> *Automatic*
+====
+
+.. In the *Details* tab of the installed Operator page, under *ClusterServiceVersion details*, verify that the installation *Status* is *Succeeded*.
+
+. Create a Secret for your object storage bucket.
++
+You can do this in the *Administrator* view of the OpenShift web console: go to *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
```

Comment:
Again, the table in the middle of the procedure is breaking the reading flow. I would suggest to move the table out of the procedure. We can place the info before this procedure so that users have the context set that while creating a secret, they must use those secret parameters.

Reply:
Thank you. I really like your suggestion. I will implement it as an improvement in a separate PR after this one.